fastlane/lib/assets/AppfileTemplate.swift

```diff
@@ -1,8 +1,7 @@
 var appIdentifier: String { return "[[APP_IDENTIFIER]]" } // The bundle identifier of your app
 var appleID: String { return "[[APPLE_ID]]" } // Your Apple email address
 
-var teamID: String { return "[[DEV_PORTAL_TEAM_ID]]" } // Developer Portal Team ID
-var itcTeam: String? { return [[ITC_TEAM]] } // iTunes Connect Team ID (may be nil if no team)
+[[TEAMS]]
```

Comment:
not valid swift

Reply:
This gets replaced during onboarding, as `[[TEAMS]]` is the placeholder

Reply:
PlaceHODLer